### ISSUE TITLE
Using new with Boom.unauthorized causes `is not a constructor` exception

### ISSUE THREAD

**Issue by miruiz** [hapijs/boom#181]
I recently upgrade boom in a project of mine and I started getting `is not a constructor` errors when using the `new` keyword.

So I created a small test project to eliminate complexity, and still get the error.

here's the entirety of my test project code...

```javascript
let b = require('boom');
throw new b.forbidden('hit the test');
```

package.json dependencies look like this:

```
"dependencies": {
    "boom": "^7.1.1"
  }
```

and the resulting error is...

```
TypeError: b.forbidden is not a constructor
```

Node version is 8.1.2

Is this expected behavior now? I thought I noticed the PR about making `new` required...

**Comment by arb** [comment]
Per the package.json file, the required node version is `>=8.9.0` Can you update and see if that makes a difference?

**Comment by miruiz** [comment]
Thanks for getting back to me! I took these steps...

1. upgraded node to latest (v9.3.0)
2. deleted my test project
3. Recreated my test project with a fresh `npm init` and fresh install of boom ^7.1.1

and I'm still seeing the error.

I did `console.log(b.forbidden);` and the output is

```
[Function: forbidden]
```

so `forbidden` should be something. I've also tried a few other error types (`unauthorized` and `serverUnavailable`), and they all give the same `not a constructor` error.

trying it again with the version of boom I was using before (^5.1.0), and it works as expected.
